docs/plugins.md

```diff
@@ -0,0 +1,34 @@
+## Plugins
+
+Use the [Auto-GPT Plugin Template](https://github.com/Significant-Gravitas/Auto-GPT-Plugin-Template) as a starting point for creating your own plugins.
```

Comment:
drop this line, and merge the plugin changes from #2883

Reply:
Or potentially just push these changes up to the main readme and reference them there?